packages/server/src/user/user.service.ts

```diff
@@ -1,94 +1,113 @@
-      project_id: 'project-001',
-      username: 'user0',
-      email: 'some.user@mail.com',
-      password: 'some_user_password',
-      role: 0,
-      created_at: null,
-      updated_at: null,
-      deleted_at: null,
-      reset_code: 'some_reset_code',
-      reset_code_expires_at: null
-    },
-    {
-      id: '94ea3787-16e0-4a5c-b9a5-d345f4fa82dd',
-      project_id: 'project-001',
-      username: 'user1',
-      email: 'another.user@mail.com',
-      password: 'another_user_password',
-      role: 0,
-      created_at: null,
-      updated_at: null,
-      deleted_at: null,
-      reset_code: null,
-      reset_code_expires_at: null
-    },
-    {
-      id: 'e929ebd5-a899-4df2-a94c-0f024de39cbd',
-      project_id: 'project-002',
-      username: 'admin0',
-      email: 'some.admin@mail.com',
-      password: 'some_admin_password',
-      role: 1,
-      created_at: null,
-      updated_at: null,
-      deleted_at: null,
-      reset_code: null,
-      reset_code_expires_at: null
+  // TODO: remove this function when deploying
+  /**
+   * This function is for creating a list of users record in database. It should be called only once.
+   * The purpose of this function is for testing only and should be removed before deployment.
+   * You can also modify or add more user obejct to the `temp_users` list.
+   */
+  async createTempUsers(): Promise<void> {
+    // TODO: add temp users to database
+    const temp_users = [
+      {
+        project_id: 'project-001',
+        username: 'admin0',
+        email: 'some.admin@mail.com',
+        password: await argon2.hash('some_admin.password'),
+        role: 1,
+        created_at: new Date('2020-01-01T09:00:01'),
+        updated_at: new Date('2020-01-01T09:00:01')
+      },
+      {
+        project_id: 'project-001',
+        username: 'user0',
+        email: 'some.user@mail.com',
+        password: await argon2.hash('some_user_password'),
+        created_at: new Date('2020-10-21T09:10:33'),
+        updated_at: new Date('2020-10-21T09:10:33')
+      },
+      {
+        project_id: 'project-001',
+        username: 'user1',
+        email: 'another.user@mail.com',
+        password: await argon2.hash('another-user-password'),
+        created_at: new Date('2021-02-17T15:44:10'),
+        updated_at: new Date('2021-12-30T12:03:01')
+      },
+      {
+        project_id: 'project-002',
+        email: 'the_admin@mail.com',
+        password: await argon2.hash('the_admin@project2'),
+        role: 3,
+        created_at: new Date('2022-02-10T10:30:00'),
+        updated_at: new Date('2022-03-01T15:32:09')
+      },
+      {
+        project_id: 'project-002',
+        email: 'one_poor_user@mail.com',
+        password: await argon2.hash('a_poor_user'),
+        created_at: new Date('2022-05-10T15:10:30'),
+        updated_at: new Date('2022-05-10T15:10:30')
+      }
+    ];
+
+    for (const user of temp_users) {
+      await this.prisma.uSER.create({
```

Comment:
`USER` instead of `uSER`?

Reply:
Need to update the scheme to be `User`, instead of `USER`. So primsa will be `this.prisma.user`